pkg/reconciler/kubernetes/tektonhub/tektonhub.go

```diff
@@ -150,6 +151,29 @@ func (r *Reconciler) FinalizeKind(ctx context.Context, original *v1alpha1.Tekton
 	if err := r.extension.Finalize(ctx, original); err != nil {
 		logger.Error("Failed to finalize platform resources", err)
 	}
+
```

Comment:
Do we need to handle all this? Can't we better do add ownerReference in job and it gets automatically handled?

Reply:
> Do we need to handle all this? Can't we better do add ownerReference in job and it gets automatically handled?

Job has already ownerReference of `db-migration` installerset and while deleting hub config cr it deletes the db-migration installerset and removes ownerReference from the job

Reply:
i think we can handle this better, see this https://github.com/kubernetes/client-go/issues/495